src/js/Progress.jsx

```diff
@@ -0,0 +1,43 @@
+var React = require('react');
+
+var Progress = React.createClass({
+  propTypes: {
+    progress: React.PropTypes.number,
```

Comment:
This is interesting. I'll create a ticket to go back to other components and add this.